bci_tester/data.py

```diff
@@ -200,8 +200,12 @@ def create_container_version_mark(
     BCI_DEVEL_REPO = f"https://updates.suse.com/SUSE/Products/SLE-BCI/{OS_MAJOR_VERSION}-SP{OS_SP_VERSION}/{LOCALHOST.system_info.arch}/product/"
     _BCI_REPLACE_REPO_CONTAINERFILE = ""
 else:
-    _BCI_REPLACE_REPO_CONTAINERFILE = f"RUN sed -i 's|baseurl.*|baseurl={BCI_DEVEL_REPO}|' /etc/zypp/repos.d/{BCI_REPO_NAME}.repo"
+    bci_repo_path = f"/etc/zypp/repos.d/{BCI_REPO_NAME}.repo"
+    # only try to replace the baseurl if the file exists and we have perl
+    # available (should be more ubiquitous than sed)
```

Comment:
We have more images with perl than with sed?

Reply:
perl is a hard dependency of rpm and thereby guaranteed to be in every container except minimal & busybox. It doesn't really matter in practice at the moment (as everything without sed also lacks zypper and thereby these tests get skipped). But it should be more future proof.